### ISSUE TITLE
writeEnv: add append option

### ISSUE THREAD

**Issue by marktani** [graphql-boilerplates/graphql-boilerplate-install#3]
Currently, `writeEnv` overwrites the file content of `.env`. An `append` option would be super useful to support existing `.env` files.

For example, in the [node-advanced boilerplate](https://github.com/graphql-boilerplates/node-graphql-server/), I use env var `APP_SECRET` that is gone after running `writeEnv`.

**Comment by kbrandwijk** [comment]
Yeah, that process should be more intelligent. Maybe using `dotenv-save`.

**Comment by marktani** [comment]
For now, I'm using `replaceInFiles(['.env], ...)` instead of `writeEnv` and it fulfills my needs 🙂 So I'm not sure if we need to change the way `writeEnv` works.

**Comment by kbrandwijk** [comment]
There's also a singular version `replaceInFile` for single file replacement by the way...
